src/Database/WorkflowInstanceRepository.cs

```diff
@@ -95,6 +95,26 @@ public async Task<bool> UpdateTaskStatusAsync(string workflowInstanceId, string
             }
         }
 
+        public async Task<bool> UpdateTaskOutputArtifactsAsync(string workflowInstanceId, string taskId, Dictionary<string, string> outputArtifacts)
+        {
+            Guard.Against.NullOrWhiteSpace(workflowInstanceId, nameof(workflowInstanceId));
+            Guard.Against.NullOrWhiteSpace(taskId, nameof(taskId));
+            Guard.Against.Null(outputArtifacts, nameof(outputArtifacts));
+
+            try
+            {
+                var result = await _workflowInstanceCollection.FindOneAndUpdateAsync(
```

Comment:
if `result` is not used, you may assign it to `_`

Reply:
removed variable